site/app/libraries/routers/WebRouter.php

```diff
@@ -83,14 +85,6 @@ public function __construct(Request $request, Core $core, $logged_in, $is_api =
     }
 
     public function run() {
-        if (is_null($this->parameters)) {
```

Comment:
They are moved to the API part for clarity - These JSON responses are for API only.